src/styles/blocks/header.scss

```diff
@@ -0,0 +1,50 @@
+.header {
+  @extend %flex-align-center-space-between;
+  padding: 0 50px;
+  height: 60px;
+
+  &__logo {
+    @include square($logo-size);
+
+  }
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+  }
+
+  &__item {
+    padding: 22.5px 0;
```

Comment:
it's redundant, you've already set paddings to a link

Reply:
if I remove it, it will become an error on tests
![image](https://user-images.githubusercontent.com/126285904/230706677-af33fde7-c0ad-4d20-8463-cd6b732e1c47.png)

Reply:
link padding is used to offset the blue underline 
item padding helps to center items